packages/amplify-cli/src/domain/amplify-usageData/UsageDataPayload.ts

```diff
@@ -14,7 +14,7 @@ export class UsageDataPayload {
   osRelease: String;
   nodeVersion: String;
   state: String;
-  isCi: Boolean | undefined;
+  isCi: Boolean;
```

Comment:
Why are you using object type instead of primitive type?
```suggestion
  isCi: boolean;
```

Reply:
I can move it to primitive type wouldn't affect the other end I believe

Reply:
It should not. But was there any reason for  not choosing primitive type?

Reply:
Not really any reason but I looked back and using primitive is better practice